srcpkgs/dispcalGUI/template

```diff
@@ -1,25 +1,34 @@
 # Template file for 'dispcalGUI'
 pkgname=dispcalGUI
-version=3.8.9.3
-revision=5
-_name="DisplayCAL"
-build_style=python2-module
-hostmakedepends="python"
-makedepends="python-devel libXxf86vm-devel libXinerama-devel libXrandr-devel"
-depends="python-dbus python-numpy wxPython argyllcms"
+version=3.9.8
+revision=1
+build_style=python3-pep517
+hostmakedepends="python3-setuptools python3-wheel"
+makedepends="python3-devel libXxf86vm-devel libXinerama-devel libXrandr-devel"
+depends="python3-distro python3-dbus python3-numpy python3-Pillow
+ python3-chromecast python3-send2trash wxPython4 python3-zeroconf
+ python3-certifi argyllcms"
 short_desc="Display Calibration and Characterization"
 maintainer="lemmi <lemmi@nerd2nerd.org>"
 license="GPL-3.0-or-later"
-homepage="https://displaycal.net/"
-changelog="https://displaycal.net/CHANGES.html"
-distfiles="${SOURCEFORGE_SITE}/dispcalgui/${_name}-${version}.tar.gz"
-checksum=7c34dfbd9f66f24f1d4c88de4a5a0de688aad719f095874b6259637d30893bea
+homepage="https://github.com/eoyilmaz/displaycal-py3"
+changelog="https://github.com/eoyilmaz/displaycal-py3/releases/tag/${version}"
+distfiles="https://github.com/eoyilmaz/displaycal-py3/archive/${version}.tar.gz"
+checksum=a9e8bba02f9bdf8df506f0dcbbea4eae48795001cb81ea5968fbfc41f4b72e1d
+# requires running X11 server and dbus-daemon
+make_check=no
+
+post_patch() {
+	# set the hardcoded version of python 3 to the latest one
+	sed -i "s|^\(py_maxversion =\).*|\1 (${py3_ver%.*}, ${py3_ver#*.})|" DisplayCAL/meta.py
```

Comment:
replace all seds with vsed to make sure we know when they stop changing anything